### ISSUE TITLE
API Enhancement

### ISSUE THREAD

**Issue by XedricKaes** [VolmitSoftware/VehiclesPlus#26]
**Is your enhancement related to a problem? Please describe.**
The problem is, that at this moment there aren't a lot off API functions. We can't properly use the API and know it will function good. Also I would like to request more functions in the API for the vehicles. 

**Describe the solution you'd like**
Check the API on outputs/inputs more often. Also do a test wether it actually works.
API functions I would like to be added/remain:
- getDrivingVehicles : A function which returns a list off all vehicles, that are being controlled by a player. This includes cars that are standing still, but are seated by any player.
- getVehicle(Player) : Get the vehicle a player is driving. If the player is not DRIVING/Passenger, return null.
- More clear usage off the getters off cars fe: getSpeed (currentspeed), getMaxSpeed, max speed as set in parameter, getAcceleration, get the acceleration the car is having at the moment. If negative, the car is slowing down. 
- Make sure that the Setters are correct, and can be used publicly. fe: if I were to set maxSpeed to 0 the car would suddenly stand stil.

*Edit: 
- Getting a car by its UUID would be a nice add-on. From there we could target specific cars.
- Getting a list off all spawned cars, (and maybe all cars from specific type/base vehicle). (Developers could filter on that themselves aswell, so 1 full list would do I guess.)
- Basically making sure we could get ALL information from a car, And edit all parameters aswell (steering, speed etc).
End Edit.*

**Additional context**

Any questions asked, I am happy to help. My Dc: [Dev] iScart#2060

**Comment by stijnb1234** [comment]
Will look into this. It contains some methods that are easy to make and usefull.
